backend/app/Models/Role.php

```diff
@@ -21,7 +21,7 @@ public function users()
 
     public function permissions()
     {
-        return $this->belongsToMany(Permission::class, 'permission_roles')
+        return $this->belongsToMany(Permission::class)
             ->withTimestamps();
```

Comment:
Also remove withTimestamps() function in role model and timestamps columns in roles table migration